### PR TITLE
Update doc: iOS does not support destination middleware

### PR DESCRIPTION
**Pull request by joeynmq**
### Proposed changes

There is a known issue where destination middlewares are not functioning well for the iOS library. The Engineering team will now be focusing on the new libraries (Swift & Kotlin) so the issue won't be fixed. 

JIRA: https://segment.atlassian.net/browse/LIBMOBILE-635

### Merge timing
ASAP once approved

### CODE REVIEW

File: src/connections/sources/catalog/libraries/mobile/ios/middleware.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ title: Middleware for iOS
...
 ---
 
-Middlewares are simple functions invoked by the Segment libraries, which give you a way to add information to the events you collect using the Segment SDKs. They can be used to monitor, modify, or reject events. Source Middleware are available on `analytics-ios` 3.6.0 and later. Destination Middleware are available on `analytics-ios` 4.0.0 and later.
+Middlewares are simple functions invoked by the Segment libraries, which give you a way to add information to the events you collect using the Segment SDKs. They can be used to monitor, modify, or reject events. Source Middlewares are available on `analytics-ios` 3.6.0 and later. 
 
 You can access the middleware API in both Objective-C and Swift.
 
 > info ""
-> **Note**: Destination-middleware only act on [data sent to destinations in device-mode](/docs/connections/destinations#connection-modes). Since the destination middleware code exists in your app or project, it cannot transform the data sent from the Segment servers to the destination endpoint. 
+> **Note**: Destination middleware is **not** available for iOS.
```
Comment (niallzato) [inline src/connections/sources/catalog/libraries/mobile/ios/middleware.md:11]:
```suggestion
> **Note**: Destination Middlewares are **not** available for iOS.
```

Comment (niallzato) [inline src/connections/sources/catalog/libraries/mobile/ios/middleware.md:11]:
```suggestion
> **Note**: Destination Middlewares are **not** available for iOS.
```

File: src/connections/sources/catalog/libraries/mobile/ios/middleware.md [hunk 2]
```diff
@@ -162,11 +157,8 @@ config.sourceMiddleware = [
     turnScreenIntoTrack,
     enforceEventTaxonomy,
     customizeAllTrackCalls,
-    blockScreenCallsToAmplitude,
-]
-config.destinationMiddleware = [
-    SEGDestinationMiddleware(key: mixpanelIntegration.key(), middleware: [sampleEventsToMixpanel]),
-    SEGDestinationMiddleware(key: amplitudeIntegration.key(), middleware: [customizeAmplitudeTrackCalls])
+    dropSpecificEvents,
+    blockScreenCallsToAmplitude
```
Comment (niallzato) [inline src/connections/sources/catalog/libraries/mobile/ios/middleware.md:161]:
```suggestion
    blockScreenCallsToAmplitude,
```

File: src/connections/sources/catalog/libraries/mobile/ios/middleware.md [hunk 3]
```diff
@@ -190,12 +182,8 @@ config.sourceMiddleware = @[
...
     enforceEventTaxonomy,
     customizeAllTrackCalls,
-    blockScreenCallsToAmplitude,
-];
-
-config.destinationMiddleware = @[
-    [[SEGDestinationMiddleware alloc] initWithKey:mixpanelIntegration.key middleware:@[sampleEventsToMixpanel]];
-    [[SEGDestinationMiddleware alloc] initWithKey:amplitudeIntegration.key middleware:@[customizeAmplitudeTrackCalls]];
+    dropSpecificEvents,
+    blockScreenCallsToAmplitude
```
Comment (niallzato) [inline src/connections/sources/catalog/libraries/mobile/ios/middleware.md:186]:
```suggestion
    blockScreenCallsToAmplitude,
```

File: src/connections/sources/catalog/libraries/mobile/ios/middleware.md [hunk 4]
```diff
@@ -367,7 +355,7 @@ SEGBlockMiddleware *dropSpecificEvents = [[SEGBlockMiddleware alloc] initWithBlo
 
 #### Block specific call types to a specific destination
 
-The following example blocks only screen calls from reaching the Amplitude destination.
+The following example blocks only screen calls from reaching the Amplitude destination.
```
Comment (niallzato) [inline src/connections/sources/catalog/libraries/mobile/ios/middleware.md:358]:
```suggestion
The following example blocks only screen calls from reaching the Amplitude destination.
```

**Review by niallzato — CHANGES_REQUESTED** [review]
Looks great! Some small changes requested.

**Review by forstisabella — APPROVED** [review]

**Comment by forstisabella** [comment]
@joeynmq does Niall need to re-review this or can I go ahead and merge?

**Comment by joeynmq** [comment]
hey @forstisabella the [KCS ticket](https://segment.atlassian.net/browse/KCS-506) has been approved, thank you!!

**Pull request merged**